lib/cronofy/auth.rb

```diff
@@ -2,28 +2,76 @@
 
 module Cronofy
   class Auth
-    API_URL = 'https://api.cronofy.com'
     APP_URL = 'https://app.cronofy.com'
+    API_URL = 'https://api.cronofy.com'
+
+    attr_reader :access_token
```

Comment:
Is this exposing more than we need to?

Looking at the call sites we're only using the `get`, `delete`, and `post` methods so perhaps `Auth` should have those to avoid exposing the whole `AuthToken` object.